egs/babel_multilang/s5/local/nnet3/run_common_langs.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+# Copyright 2016 Pegah Ghahremani
+
+# This script used to generate MFCC+pitch features for input language lang.
+
+. ./cmd.sh
+set -e
+stage=1
+train_stage=-10
+generate_alignments=true # If true, it regenerates alignments.
+speed_perturb=true
+use_pitch=true      # If true, it generates pitch features and combine it with 40dim MFCC.
+pitch_conf=conf/pitch.conf # Configuration used for pitch extraction.
+use_pitch_plp=false # If true, it generated plp+pitch to be used in regenerating alignments.
```

Comment:
Pegah, I think the use_pitch vs use_pitch_plp as it is now is not completely right.
The thing is that use_pitch is used in the babel recipes to use/not use pitch with plp (i.e. the same meaning as your use_pitch_plp) so you don't use the babel's configuration but by sourcing common_vars.sh it seems you are. Also, aligning of train_sp will fail (because the values of babel's ue_pitch and your use_putch_plp are not synchronized.